### DISCUSSION TITLE
Adding a List of Car Schema to ItemList's ItemListElement property give implicit type convert error

### DISCUSSION THREAD

**Discussion by naveadkazi** [RehanSaeed/Schema.NET#140] [Q&A]
### Question

I am trying to create a schema for a list of cars. To achieve this, I am trying to add a List<Car> to an ItemList.ItemListElement object. This giving following error 
`
Cannot implicitly convert type 'System.Collections.Generic.List<Schema.NET.Car>' to 'Schema.NET.Values<Schema.NET.IListItem, string, Schema.NET.IThing>'
`

Below is the code 
``` C#
private ItemList CreateCarSchema(CarResult carResult)
        {
            ItemList itemList= new ItemList();
            List<Car> cars = new List<Car>();
            foreach (var result in carResult)
            {
                var car = new Car
                {
                    Name = result.CarName
                };
                cars.Add(car);
            }
            itemList.ItemListElement = cars;
            return itemList;
        }
```
Did i miss something here?
### Schema Objects

Which [schema.org](https://schema.org) object is this about if any?

https://schema.org/Car
https://schema.org/ItemList

**Comment by RehanSaeed** [comment]
See the unit tests for an example:

https://github.com/RehanSaeed/Schema.NET/blob/9ff6b23f0425fd011fd2f565b3090f0848a08396/Tests/Schema.NET.Test/core/BreadcrumbListTest.cs

**Comment by naveadkazi** [comment]
Thanks for the example. This solved the issue.
